client/Makefile

```diff
@@ -38,7 +38,7 @@ smoke-test:
 	start_server_and_test \
 		'CXG_OPTIONS="--config-file $(CXG_CONFIG)" $(MAKE) start-server' \
 		$(CXG_SERVER_PORT) \
-		'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e -- --verbose false'
+		'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e -- --verbose --no-cache false'
```

Comment:
this helps my dev process. Otherwise updates don't get included. Not an npm expert. 

Is this going to slow things down in our ci/cd pipeline?

Reply:
which updates dont get included? front end or backend or both?

Reply:
FE (changes to jest tests)